src/FaceGenManager.cpp

```diff
@@ -75,18 +88,176 @@ void FaceGenManager::InstallIgnorePreprocessedFaceGen()
 	logger::info("Installed hooks for ignoring preprocessed FaceGen");
 }
 
+void FaceGenManager::SkipHeadPartsValidation()
+{
+	static REL::Relocation<std::uintptr_t> hook_verify{ Offset::TESNPC_LoadFaceGen, 0x4A9 };
+
+	REL::safe_write(hook_verify.address(), nop6);
```

Comment:
Logging trampoline should maybe go here as well. At the moment it wouldn't log anything when the head parts list doesn't match.